backend/api/Models/Concepts/Note/NoteModel.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 
-namespace Pims.Api.Areas.Notes.Models
+namespace Pims.Api.Models.Concepts
 {
     public class NoteModel
```

Comment:
Please extend from BaseAppModel instead of adding AppLastUpdateUserid to be class

Reply:
We are also missing **RowVersion** here - that will become important for editing notes later